includes/class-wc-gateway-amazon-payments-advanced.php

```diff
@@ -185,6 +185,172 @@ public function display_amazon_pay_button_separator_html() {
 		<?php
 	}
 
+	public function maybe_create_index_table() {
```

Comment:
This is a maybe, but looks like it always create the table, I do not see the condition of not to do it.

Reply:
If you call dbDelta with false as a second argument, it won't actually create the table, it'll just calculate the queries to be performed. 

I know there are better ways to do this, but at alpha stage, we can get away with this. 

Should I add a TODO comment on this function to be able to get back to this?

Reply:
Added a TODO note